Лабораторная 1

TriPechenki opened the pull request.

TriPechenki committed:
Лабораторная 1

frek17 commented on Лабораторная 1/Лабораторная работа 1/task_1.py:
```diff
@@ -0,0 +1,211 @@
+        :param zoom: Увеличение при съемке
+
+        Пример:
+        >>> photo = PhonePhoto("selfie", 7)
```
А еще есть режимы, кроме селфи? Будем проверять, что режим существует?

TriPechenki replied:
Поняла, все поправлю/допишу!

frek17 commented on Лабораторная 1/Лабораторная работа 1/task_1.py:
```diff
@@ -0,0 +1,211 @@
+class CoffeeСup:
+    def __init__(self, shot_of_espresso: int):
+        """
+        Создание и подготовка к работе объекта "Кофе"
```
Так кофе или чашечка кофе?

frek17 commented on Лабораторная 1/Лабораторная работа 1/task_1.py:
```diff
@@ -0,0 +1,211 @@
+        :param shot_of_espresso: Кол-во желаемых шотов
+
+        Пример:
+        >>> coffee = CoffeeСup(2)
```
Обычно экземпляр класса дублирует название класса, поэтому давайте назовём класс `Coffee`

TriPechenki committed:
Лабораторная работа 1 исправление

- Откорректированы возможные режимы съемки в классе PhonePhoto
- Исправлены недочеты в классе Coffe

frek17 commented:
Очень круто получилось! Возможно, эта работа больше всех понравилась и в плане креатива и в плане выполнения самого задания
